austrakka/components/sequence/funcs.py

```diff
@@ -523,12 +524,36 @@ def _get_files_from_csv_paths(csv_row: pd.Series, seq_type: SeqType) -> List[Seq
         if column==SEQ_ID_CSV:
             continue
         contig_rename_id = seq_id if seq_type==SeqType.FASTA_ASM else None
-        file = _get_file(csv_row[column], contig_rename_id)
+        read_hint = _col_name_to_read_hint(column)
+        file = _get_file(csv_row[column], contig_rename_id, read_hint)
+        _ensure_read_hint_and_filename_agrees(file)
         sample_files.append(file)
     return sample_files
 
 
-def _get_file(filepath: str, contig_rename_id=None) -> SeqFile:
+def _col_name_to_read_hint(column):
```

Comment:
This is fine but seems unnecessary - could just assign the column name as the "hint"?